storage/innobase/fil/fil0fil.cc

```diff
@@ -10671,22 +10654,12 @@ const byte *fil_tablespace_redo_rename(const byte *ptr, const byte *end,
     return nullptr;
   }
 
-<<<<<<< HEAD
-  ptr += to_len;
-  Fil_path::normalize(to_name);
-
```

Comment:
we have special code with PXB. if parse_only, return ptr.

In the old code, there is ptr is incrementd by to_len. With the new code, there is no to_len. Is the ptr moved else where?

Reply:
yes this logic is moved to parse_path_from_redo()